server/schedule/replica_checker.go

```diff
@@ -251,3 +232,33 @@ func (r *ReplicaChecker) checkBestReplacement(region *core.RegionInfo) *Operator
 	checkerCounter.WithLabelValues("replica_checker", "new_operator").Inc()
 	return CreateMovePeerOperator("moveToBetterLocation", r.cluster, region, OpReplica, oldPeer.GetStoreId(), newPeer.GetStoreId(), newPeer.GetId())
 }
+
+func (r *ReplicaChecker) fixPeer(region *core.RegionInfo, peer *metapb.Peer, status string) *Operator {
```

Comment:
what does fixPeer mean?

Reply:
It means this peer needs to be removed in order to be made up in another store or just simply be replaced. The name before addressing the comment is `handleReplica`. There could be a better naming about this.